lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+- Complete the *first* section on @printable-exercise{graphing-models.adoc}.
+- Using `lr-plot` in Pyret, come up with the best possible linear model, and write your answer at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are linear models a good fit for this data? Why or why not?
+
+Linear models capture _straight-line relationships_, where one quantity varies proportionally based on another. In linear models, we expect the response variable to grow by equal amounts over equal intervals in the explanatory variable.
+
+@right{@image{images/MA-covid.png}} If we make the line go from the start to the peak, almost all of the points bulge out below out line of best fit. If we make the line fit the bulge, all the points fall above it! We always have either too many points _below_ the line in the middle or _above_ the line at the end. **It's growing too fast to be fit with a linear model that grows at a constant rate!**
+
+[.lesson-instruction]
+- Take a few minutes to explore @online-exercise{https://www.desmos.com/calculator/73y2med4yj, MA Covid Spread - Quadratic}.
+- Complete the *second* section on @printable-exercise{graphing-models.adoc}.
+- Write the best quadratic model you can find at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are quadratic models a good fit for this data? Why or why not?
+
+Have students share their resulting models. Which one fits best?
+
+Quadratic models capture _parabolic relationships_, where one quantity varies based on the square of another. In quadratic models, we expect the response variable to grow by differing amounts over equal intervals in the explanatory variable.
+
+@right{@image{images/MA-covid.png}} Quadratic models change their rate of growth over time, which definitely made them a better fit than linear ones. But they still don't have the explosive growth we need to model this data, which starts out incredibly slow and then suddenly takes off like a rocket!
+
+There is, however, a class of functions that grow even faster than quadratics: *exponential functions*.
+
+Exponential relationships show up all the time!
+
+- Cells that constantly divide, doubling the total number of cells each time
+- A tree that's on fire will likely catch neighboring trees on fire, so that the fire spreads to 2 or more trees each time
+- Money in a savings account grows by a certain percentage each year, meaning every year there's that much more money to grow
+
+[.lesson-point]
+Linear functions grow by equal *amounts* over equal intervals (adding _n_ each time). Exponential functions grow by equal *factors* over equal intervals (multiplying by _n_ each time).
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely exponential?
+
+Exponential models have the form @math{y = ab^x + k}
+
```

Comment:
I would add, "As you discovered while exploring desmos, a is the initial value, b is a growth factor, and k is the vertical shift."

(You probably want to elaborate more on these.)